### PR TITLE
Container.rename + unit tests

### PR DESCRIPTION
**Pull request by amihaiemil**
PR for #97

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#102](https://github.com/amihaiemil/docker-java-api/issues/102) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#102](https://github.com/amihaiemil/docker-java-api/issues/102) is assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/1301330081); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

**Comment by 0crat** [comment]
Job [#102](https://github.com/amihaiemil/docker-java-api/issues/102) is already in scope

File: src/main/java/com/amihaiemil/docker/Container.java [hunk 1]
```diff
@@ -81,4 +81,16 @@
      *     expected.
      */
     void restart() throws IOException, UnexpectedResponseException;
+    
+    /**
+     * Rename this container.
+     * @param name New name for the container.
+     * @throws IOException If something goes wrong.
+     * @throws UnexpectedResponseException If the status response is not
+     *     expected.
```
Comment (llorllale) [inline src/main/java/com/amihaiemil/docker/Container.java]: @amihaiemil you don't want to include a link to the docker API docs?

**Comment by llorllale** [comment]
@amihaiemil I guess we're good here?

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31):

**Comment by 0crat** [comment]
The job [#102](https://github.com/amihaiemil/docker-java-api/issues/102) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Pull request merged**